.github/workflows/linter.yml

```diff
@@ -5,27 +5,37 @@ on:
   - push
 
 jobs:
-  build:
-    name: Lint OpenAPI Descriptions
+  lint:
+    name: Lint releases
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Code
         uses: actions/checkout@v4
-      - name: Yaml Lint
-        uses: karancode/yamllint-github-action@v2.0.0
+      - uses: actions/setup-python@v5
+        name: Install Python
         with:
-          yamllint_file_or_dir: 'descriptions/**/*.yaml'
-          yamllint_strict: false
-          yamllint_config_filepath: '.yamllint-config.yml'
-          yamllint_comment: true
-      - uses: actions/setup-node@v3
+          python-version: '3.12'
+      - run: pip install yamllint
+        name: Install yamllint
+      - run: |
+          echo "current directory $(pwd)"
+          echo "current directory contents"
+          ls -l .
+          echo "parent directory contents"
+          ls -l ..
+          yamllint -f parsable -c .yamllint-config.yml descriptions/api.github.com/*.yaml
```

Comment:
What about the 3.1 descriptions?

Reply:
I was going to hold off on that and run both workflows in matrix-mode when the redocly action is ready.

But I could have a go at enabling that for YAML linting while we're here...

Reply:
9e3a5b8c809241f020946b99c7a1ac1b38fa1a79